src/System.Net.Requests/src/System/Net/WebRequest.cs

```diff
@@ -370,13 +370,15 @@ internal static List<WebRequestPrefixElement> PrefixList
                         if (s_prefixList == null)
                         {
                             var httpRequestCreator = new HttpRequestCreator();
+                            var ftpRequestCreator = new FtpWebRequestCreator();
                             var fileRequestCreator = new FileWebRequestCreator();
 
-                            const int Count = 3;
+                            const int Count = 4;
                             var prefixList = new List<WebRequestPrefixElement>(Count)
                             {
                                 new WebRequestPrefixElement("http:", httpRequestCreator),
                                 new WebRequestPrefixElement("https:", httpRequestCreator),
+                                new WebRequestPrefixElement("ftp:", ftpRequestCreator),
```

Comment:
We support TLS upgrade via the AUTH TLS command.

We don't support ftps:, desktop doesn't either.